Content.Shared/Trigger/Components/Effects/AddReagentOnTriggerComponent.cs

```diff
@@ -0,0 +1,27 @@
+using Content.Shared.Chemistry.EntitySystems;
+using Robust.Shared.GameStates;
+using Content.Shared.Chemistry.Components;
+
+namespace Content.Shared.Trigger.Components.Effects;
+
+[RegisterComponent, AutoGenerateComponentState]
+public sealed partial class AddReagentOnTriggerComponent : BaseXOnTriggerComponent
+{
+    /// <summary>
+    /// The name of the solution to add to.
+    /// </summary>
+    [DataField("solution", required: true)]
+    public string SolutionName = string.Empty;
```

Comment:
```suggestion
    [DataField(required: true)]
    public string SolutionId = string.Empty;
```
just keep the C# and yaml name the same